pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -107,12 +108,19 @@ func InstallPlugin(pluginName, version string, c utils.CommandLine) error {
 	logger.Infof("into: %v\n", pluginFolder)
 	logger.Info("\n")
 
-	content, err := c.ApiClient().DownloadFile(pluginName, pluginFolder, downloadURL, checksum)
+	// Create temp file for downloading zip file
+	tmpFile, err := ioutil.TempFile("", "*.zip")
+	if err != nil {
+		errutil.Wrap("Failed to create temporary file", err)
```

Comment:
Do you mean `return errutil.Wrap("Failed to create temporary file", err)`?

Reply:
Of course, thanks!